network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/tito_widget_block.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./base_streamfield_block.html" %}
+
+{% block block_content %}
+    <script src='https://js.tito.io/v2' async></script>
+
+    {% if value.redirect_page %}
```

Comment:
Use of redirect page to-be-confirmed

Reply:
If there's follow-up investigation to be done, please remember to file an issue before this PR lands.